etc/docker/php/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+chown -R www-data:www-data -R app/cache app/logs
```

Comment:
I heard that changing permissions in the entrypoint makes the Docker taking twice the space, is that correct in this case? These directories may not be empty.